src/rotations.jl

```diff
@@ -1,46 +0,0 @@
-# ------------------------------------------------------------------
-# Licensed under the MIT License. See LICENSE in the project root.
-# ------------------------------------------------------------------
-
-# -------------
-# 2D ROTATIONS
-# -------------
-
-# temporary fix for https://github.com/JuliaSpace/ReferenceFrameRotations.jl/issues/22 
```

Comment:
I'm understanding `ClockwiseAngle` and `CounterClockwiseAngle` are a temporary fix for 2-dimensional rotation, and can be replaced with `Rotations.Angle2d`.